--- a/spec/services/ikea_lv_product_variants_service_spec.rb
+++ b/spec/services/ikea_lv_product_variants_service_spec.rb
@@ -3,7 +3,9 @@
 require "rails_helper"
 
 RSpec.describe IkeaLvProductVariantsService do
-  let(:product) { instance_double(Product, sku: "s29545213", id: 1, name_ru: "VIMLE") }
+  let(:product) do
+    instance_double(Product, sku: "s29545213", id: 1, name: "VIMLE", name_ru: "VIMLE")
+  end
   let(:service) { described_class.new(product: product, force: true) }
 
   before do

